in_toto/model.go

```diff
@@ -750,11 +822,11 @@ func (mb *Metablock) Load(path string) (err error) {
 		mb.Signed = layout
 
 	} else {
-		return fmt.Errorf("The '_type' field of the 'signed' part of in-toto" +
+		return fmt.Errorf("the '_type' field of the 'signed' part of in-toto" +
 			" metadata must be one of 'link' or 'layout'")
 	}
 
-	return nil
+	return jsonFile.Close()
```

Comment:
Shouldn't that jsonFile.Close() be called anyway due to the defer jsonFile.Close() at beginning? Wouldn't we call it twice?

Reply:
I see this "defer early & return close"-construct the first time :D if this works, I learned something new. Cool idea.

Reply:
It would be called twice in the happy path, yes.  One of my colleagues wrote this bit here and it somewhat raised my eyebrows as well.  I believe the thinking is that if an error occurred in one of the calls in this function such as `checkRequiredJSONFields` and an error also occurred in the deferred `Close`, the deferred close would overwrite the error from `checkRequiredJSONFields`.